lib/msf/ui/tip.rb

```diff
@@ -0,0 +1,38 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Module that contains some most excellent tips.
+    #
+    ###
+    module Tip
+      def self.highlight(string)
+        "%grn#{string}%clr"
+      end
+
+      ALL = [
+        "Enable verbose logging with #{highlight('set verbose true')}",
+        "View useful productivity tips with the #{highlight('tip')} highlight, or view them all with #{highlight('tip -l')}",
+        "When in a module, use #{highlight('back')} to unload the module",
```

Comment:
This doesn't actually unload the module but rather stops using it and goes back to the top-level prompt. It's never as useful as the `previous` command, which lets you go to the previously used module.